build_projects/dotnet-host-build/PackageTargets.cs

```diff
@@ -266,5 +269,25 @@ private static void FixPermissions(string directory)
                 FS.FixModeFlags(directory);
             }
         }
+
+        private static string GetSharedFrameworkPublishPath()
```

Comment:
I guess it wasn't clear to me that you were making changes to the build system itself, when you said packaging, I interpreted that as the stuff under the pkg folder.  Most of this will be going away in the next week or two.  What's the priority on getting this done?  Would it make sense to wait on this, or to make parallel changes in my fork based on buildtools?